src/crm/debug.ts

```diff
@@ -1,98 +1,36 @@
-import _ from 'lodash';
+import _ from "lodash";
+import {
+  CRM,
+  CRMType,
+  ActionMessage,
+  handleResult,
+  ProcaCampaign,
+} from "../crm";
+import { writeFileSync } from "fs";
 
-import {writeFileSync}  from 'fs';
 /*
- * A test CRM
+ * A debug CRM that saves the data into the data folder
  *
- *
- * Holds a basic Contacts, who have ids of campaigns they signed
- * Holds basic Campaign with a name
  */
 
-
+// TODO export from proca/queue
 type ContactID = string | number | undefined;
```

Comment:
what is the use of this? Is this a CRM id of created record ?

Reply:
good catch, garbage indeed. removed.